Themes/default/Display.template.php

```diff
@@ -794,7 +794,7 @@ function template_single_post($message)
 	}
 
 	// And stuff below the attachments.
-	if ($context['can_report_moderator'] || !empty($context['can_see_likes']) || !empty($context['can_like']) || $message['can_approve'] || $message['can_unapprove'] || $context['can_reply'] || $message['can_modify'] || $message['can_remove'] || $context['can_split'] || $context['can_restore_msg'] || $context['can_quote'])
+	if ($context['can_report_moderator'] || !empty($context['can_like']) || $message['can_approve'] || $message['can_unapprove'] || $context['can_reply'] || $message['can_modify'] || $message['can_remove'] || $context['can_split'] || $context['can_restore_msg'] || $context['can_quote'])
```

Comment:
Here and in the similar statement below to close the div, you should probably replace `!empty($context['can_see_likes'])` with `!empty($modSettings['enable_likes'])`, rather than just deleting `!empty($context['can_see_likes'])`.

Reply:
Not everybody can like though...  What about folks in groups who don't have those permissions?

Reply:
Oh, I see it...  Will change & test, give me a few...

Reply:
Good catch!